Empire/ResourcePool.h

```diff
@@ -69,8 +70,8 @@ class FO_COMMON_API ResourcePool {
     std::set<std::set<int>>                 m_connected_system_groups;                          ///< sets of systems between and in which objects can share this pool's resource
     std::map<std::set<int>, float>          m_connected_object_groups_resource_output;          ///< cached map from connected group of objects that can share resources, to how much resource is output by ResourceCenters in the group.  regenerated during update from other state information.
     std::map<std::set<int>, float>          m_connected_object_groups_resource_target_output;   ///< cached map from connected group of objects that can share resources, to how much resource would, if all meters equaled their target meters, be output by ResourceCenters in the group.  regenerated during update from other state information.
-    int                                     m_stockpile_object_id;                              ///< object id where stockpile for this pool is located
-    float                                   m_stockpile;                                        ///< current stockpiled amount of resource
+    int                                     m_stockpile_object_id = INVALID_OBJECT_ID;          ///< object id where stockpile for this pool is located
+    float                                   m_stockpile = 0.0;                                  ///< current stockpiled amount of resource
```

Comment:
Yeah, I was copying over the existing values (which were in the initializer list, as you can see in the cpp file), and wasn't sure if I should make the change... I will make the change now.